web/client/plugins/DrawerMenu.jsx

```diff
@@ -20,7 +20,8 @@ const {Button, Glyphicon, Panel} = require('react-bootstrap');
 const Section = require('./drawer/Section');
 
 const {partialRight} = require('lodash');
-
+const url = require('url');
+const query = url.parse(window.location.href, true).query;
```

Comment:
I don't like plugins directly depending from the request. We should use dynamic plugin configuration of plugins to:
 * add dynamicity to hide (currently it doesn't support expressions)
 * add request params to properties available in expressions (e.g. {request.forceDrawer})